pegasus/sites.v3/code.org/public/pd-workshop-survey/splat.haml

```diff
@@ -4,6 +4,8 @@ title: "Professional Development Workshop Survey"
 :ruby
   dont_cache
   enrollment_code = request.splat_path_info[1..-1]
+  redirect (request.scheme || 'http') + ':' + CDO.studio_url("/pd/workshop_survey/#{enrollment_code}")
```

Comment:
Are you saying it doesn't with the way I suggested, but it does work the way it is, or it doesn't work in either case?